supply/lib/supply/options.rb

```diff
@@ -116,6 +117,35 @@ def self.available_options
                                          UI.user_error!("Could not parse service account json  JSON::ParseError")
                                        end
                                      end),
+        FastlaneCore::ConfigItem.new(key: :json_token,
```

Comment:
I think it make sense to call this `refresh_token`. `json_token` is a bit to similar to `json_key` and I think it would cause some confusion 😊